python/lsst/ctrl/oods/fileIngester.py

```diff
@@ -287,38 +269,14 @@ def ingest_file(self, butlerProxy, msg):
         """
 
         # get the locally staged file name
-        camera = msg['CAMERA']
-        obsid = msg['OBSID']
         filename = self.get_locally_staged_filename(butlerProxy, os.path.realpath(msg['FILENAME']))
         archiver = msg['ARCHIVER']
 
         # attempt to ingest the file;  if ingests, log that
         # if it does not ingest, move it to a "bad file" directory
         # and log that.
-        try:
-            butler = butlerProxy.getButler()
-            butler.ingest(filename)
-            LOGGER.info(f"{butler.getName()}: {obsid} {filename} ingested from {camera} by {archiver}")
-        except Exception as e:
-            status_code = self.FAILURE
-            status_msg = f"{butler.getName()}: {filename} could not be ingested: {self.extract_cause(e)}"
-            LOGGER.exception(status_msg)
-            bad_dir = butlerProxy.getBadFileDirectory()
-            staging_dir = butlerProxy.getStagingDirectory()
-
-            bad_file_dir = Utils.create_bad_dirname(bad_dir, staging_dir, filename)
-            try:
-                LOGGER.info(f"Moving {filename} to {bad_file_dir}")
-                shutil.move(filename, bad_file_dir)
-            except Exception as fmException:
-                LOGGER.info(f"Failed to move {filename} to {bad_file_dir} {fmException}")
-
-            return (status_code, status_msg)
-
-        status_code = self.SUCCESS
-        status_msg = f"{butler.getName()}: OBSID {obsid} - File {filename} ingested into OODS"
-
-        return (status_code, status_msg)
+        butler = butlerProxy.getButler()
+        butler.ingest(archiver, filename)
```

Comment:
Again, it looks to me that the comment above is outdated.

Reply:
Comment fixed